pod/perldelta.pod

```diff
@@ -428,6 +428,27 @@ manager will later use a regex to expand these into links.
 
 XXX
 
+=item * C<&CORE::__CLASS__> no longer returns filenames
+
+C<CORE::__CLASS__> would work as expected when used as a bareword or aliased:
+
+    use feature qw(class);
+    class Foo {
+        BEGIN { *cls = \&CORE::__CLASS__; }
+        method bar() {
+            my $class1 = CORE::__CLASS__;  # ok
+            my $class2 = cls;              # ok
+        }
+    }
+
+But when called with an ampersand (C<&CORE::__CLASS__()>) or through a
+reference (C<< my $ref = \&CORE::__CLASS__; $ref->() >>), it would return the
+current filename instead. These runtime calls now throw an error of the form
```

Comment:
That wasn't the intended behaviour ;) I'd not mention that and simply put something like

> it would return invalid results.

Otherwise we can almost guess someone will interpret this as documenting that it definitely *can* be used to get the filename on older perls.